pom.xml

```diff
@@ -234,6 +234,12 @@
         <version>1.7.0</version>
         <optional>true</optional>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-annotations</artifactId>
+        <version>2.13.3</version>
```

Comment:
```
<scope>test</scope>
```

let's clarify that it's only a test-scoped dependency here. the client explicitly uses gson instead of jackson to do the serialization.